magefile.go

```diff
@@ -308,8 +308,8 @@ func UpgradeOperator(ctx context.Context) error {
 	if err := sh.RunV(
 		"helm",
 		"upgrade",
-		"kedahttp",
-		"./charts/keda-http-operator",
+		"http-add-on",
+		"kedacore/http-add-on",
```

Comment:
@v-shenoy good catch - could we make the location of the chart configurable via an environment var?

Reply:
Good idea!